docs/content/manual/manual.yml

```diff
@@ -104,74 +104,58 @@ sections:
       You can affect how jq reads and writes its input and output
       using some command-line options:
 
-      * `--version`/`-V`:
```

Comment:
You don't think `--version` should be first?

Reply:
I think the option (at least the long option) is obvious and don't think it should be first.